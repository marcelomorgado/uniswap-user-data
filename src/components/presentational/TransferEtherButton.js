import React from "react";
import PropTypes from "prop-types";
import Button from "@material-ui/core/Button";
import { withStyles } from "@material-ui/core/styles";

const styles = theme => ({
  button: {
    margin: theme.spacing.unit,
  },
});

function TransferEtherButton({ onClick, classes }) {
  return (
    <Button
      variant="contained"
      color="primary"
      className={classes.button}
      onClick={onClick}
    >
      Transfer 1 ETH
    </Button>
  );
}

TransferEtherButton.propTypes = {
  classes: PropTypes.object.isRequired,
  onClick: PropTypes.func.isRequired,
};

export default withStyles(styles)(TransferEtherButton);
