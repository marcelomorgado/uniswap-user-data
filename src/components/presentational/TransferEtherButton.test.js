import React from "react";
import { shallow } from "enzyme";
import TransferEtherButton from "./TransferEtherButton";

it("renders the component", () => {
  const props = {
    onClick: () => {},
  };

  expect(shallow(<TransferEtherButton {...props} />)).toMatchSnapshot();
});
