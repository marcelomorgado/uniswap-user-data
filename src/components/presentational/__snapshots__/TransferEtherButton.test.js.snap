// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the component 1`] = `
<TransferEtherButton
  classes={
    Object {
      "button": "TransferEtherButton-button-1",
    }
  }
  onClick={[Function]}
/>
`;
